runner/src/internal.rs

```diff
@@ -97,24 +97,76 @@ pub struct BuildServer {
     pub coverage: bool,
 }
 
+#[derive(Clone)]
```

Comment:
I think we usually keep these annotations after the comment, near the actual struct?